script/ethereum/DeployMorphoTokenEthereum.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "../../lib/forge-std/src/Script.sol";
+import "../../lib/forge-std/src/console.sol";
+
+import {MorphoTokenEthereum} from "../../src/MorphoTokenEthereum.sol";
+import {Wrapper} from "../../src/Wrapper.sol";
+import {ERC1967Proxy} from
+    "../../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import {DeploymentImplementation} from "../helpers/DeploymentImplementation.sol";
+
+contract DeployMorphoTokenEthereum is Script {
+    address public constant MORPHO_DAO = 0xcBa28b38103307Ec8dA98377ffF9816C164f9AFa;
+
+    bytes32 public DEPLOYMENT_IMPLEMENTATION_SALT;
+    bytes32 public IMPLEMENTATION_SALT;
```

Comment:
No need to use CREATE2 for the implementation, as we don't care about having a predictable address for the implementation

Reply:
now that it's what has been used, let's keep it like that